addons/product_images/models/ir_cron_trigger.py

```diff
@@ -0,0 +1,26 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import _, api, models
+from odoo.exceptions import ValidationError
+
+
+class ir_cron_trigger(models.Model):
+    _inherit = "ir.cron.trigger"
+
+    @api.constrains('cron_id')
+    def _check_cron_id(self):
+        """ Check if `ir_cron_fetch_image` is already triggered
+
+        Ensure that there is a maximum of one trigger at a time for `ir_cron_fetch_image`.
+
+        :return: None
+        :raises ValidationError: If there is already an `ir_cron_fetch_image` triggered
+        """
+        ir_cron_fetch_image_id = self.env.ref('product_images.ir_cron_fetch_image').id
+        if self.cron_id.id != ir_cron_fetch_image_id:
+            return
+        nb_cron = self.env['ir.cron.trigger'].search_count(
```

Comment:
`nb_cron_triggers`?

Reply:
I'll go for `nb_cron_triggered` if that's ok with you